tests/libres_tests/res/enkf/test_local_config.py

```diff
@@ -119,14 +119,14 @@ def test_all_active(self):
             main = test_context.getErt()
 
             local_config = main.getLocalConfig()
-
+            print("Have returned local config")
             updatestep = local_config.getUpdatestep()
-            ministep = updatestep[0]
-            self.assertEqual(1, ministep.numActiveData())
-            self.assertTrue(ministep.hasActiveData("PERLIN_PARAM"))
+            # ministep = updatestep[0]
+            # self.assertEqual(1, ministep.numActiveData())
+            # self.assertTrue(ministep.hasActiveData("PERLIN_PARAM"))
 
-            obsdata = ministep.getLocalObsData()
-            self.assertEqual(len(obsdata), 3)
+            # obsdata = ministep.getLocalObsData()
+            # self.assertEqual(len(obsdata), 3)
```

Comment:
Is this no longer valid?

Reply:
Hmmm - no that should still be valid; have reverted the diff.